Source/swiftlint/Extensions/LinterCache+CommandLine.swift

```diff
@@ -45,15 +45,15 @@ private func cacheURL(options: LintOptions, configuration: Configuration) -> URL
 }
 
 private func defaultCacheURL(options: LintOptions) -> URL {
-    let rootPath = options.path.bridge().absolutePathRepresentation()
+    let cacheSignature = options.paths.map { $0.bridge().absolutePathRepresentation() }.joined(separator: "")
```

Comment:
`""` is the default separator, you should use just `.joined()` (see https://github.com/realm/SwiftLint/issues/1093)